src/RA_Defs.h

```diff
@@ -159,15 +165,22 @@ class ResizeContent
     };
 
 public:
-    HWND hwnd;
-    POINT pLT;
-    POINT pRB;
-    AlignType nAlignType;
-    int nDistanceX;
-    int nDistanceY;
-    bool bResize;
-
-    ResizeContent(HWND parentHwnd, HWND contentHwnd, AlignType newAlignType, bool isResize)
+    HWND hwnd{ nullptr };
+    POINT pLT = POINT{};
+    POINT pRB = POINT{};
+    AlignType nAlignType = AlignType{};
+    int nDistanceX{ 0 };
+    int nDistanceY{ 0 };
+    bool bResize{ false };
+
+    inline constexpr ResizeContent() noexcept = default;
+    ResizeContent(const ResizeContent& rhs) = delete;
+    ResizeContent& operator=(const ResizeContent& rhs) = delete;
```

Comment:
Copy-Epsilon optimization, didn't see any need to copy it where this was referenced but you still need to move it. Other classes share this concept.